### PR TITLE
A-10282: update nodejs version

### PR DESCRIPTION
**Pull request by shanet**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -134,11 +134,11 @@ ENV JEST_JUNIT_OUTPUT "/tmp/jest/junit.xml"
 ENV RAILS_ENV test
 ENV LC_ALL "en_US.UTF-8"
 
+# Add npmrc template.
+RUN mkdir -p /usr/etc && echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > /usr/etc/npmrc
```
Comment (shanet) [inline Dockerfile:138]: I don't know why, but this step kept resulting in:

```
`Step 67/68 : RUN mkdir -p /usr/etc && echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > /usr/etc/npmrc error creating overlay mount to overlay2/d3e214b4ce19a76299b3d2958645d826b41160ac925dd762b913999a53f89f7f-init/merged: no such file or directory`
```
during the build. Moving it above the start script steps resolved it.

File: Dockerfile [hunk 2]
```diff
@@ -55,7 +55,7 @@ RUN apt-key adv --refresh-keys --keyserver ha.pool.sks-keyservers.net
 RUN apt-get update
 
 # Install nodejs
-RUN apt-get install nodejs npm yarn -y
```
Comment (shanet) [inline Dockerfile:58]: NPM is included in the nodejs package now.

**Comment by sbeckeriv** [comment]
odd question. does building it update the chrome?

**Comment by shanet** [comment]
Yup, we're installing Chrome via `RUN apt-get install google-chrome-stable -y` so building a new package installs the most recent version, 83.

**Pull request merged**